src/test/kotlin/apply/application/UserAuthenticationServiceTest.kt

```diff
@@ -49,34 +49,60 @@ internal class UserAuthenticationServiceTest {
         )
     }
 
-    @DisplayName("토큰 생성은")
+    @DisplayName("(회원 가입) 토큰 생성은")
     @Nested
-    inner class GenerateToken {
+    inner class GenerateTokenByRegister {
         private lateinit var request: RegisterUserRequest
 
         fun subject(): String {
-            return userAuthenticationService.generateToken(request)
+            return userAuthenticationService.generateTokenByRegister(request)
         }
 
         @Test
-        fun `회원이 존재하고 인증에 성공하면 유효한 토큰을 반환한다`() {
-            every { userRepository.findByEmail(any()) } answers { createUser() }
-            request = RegisterUserRequest(NAME, EMAIL, PHONE_NUMBER, GENDER, BIRTHDAY, PASSWORD)
-            assertThat(subject()).isEqualTo(VALID_TOKEN)
+        fun `가입된 이메일이라면 예외가 발생한다`() {
+            every { userRepository.existsByEmail(any()) } answers { true }
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/woowacourse/service-apply/pull/251#discussion_r669606182

```suggestion
            every { userRepository.existsByEmail(any()) } returns true
```

Reply:
앗 요부분을 놓쳤네요 ㅎㅎㅎ 
전반적으로 반영했습니다 !